new/LIT7073QeneTamayta.xml

```diff
@@ -0,0 +1,80 @@
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript, consisting of ten lines. 
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD"/> in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-09-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7073QeneTamayta" passive="UppQ1524o"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 38r, 39r. 
+                    Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1">ተመይጠት፡ ጽዮን፡ ጼዋ፡ አጵሎን፡ ነኪር፤</l>
+                    <l n="2">ሀገሮ፡ ለወልድ፡ መስተገብር፤</l>
+                    <l n="3">ገሞራ፡ ድህረ፡ ነጸረ፡ በዓይነ፡ አጵሎን፡ ገብር፤</l>
+                    <l n="4">እምነ፡ ሀገሩ፡ ዘወትረ፡ መአር፤</l>
+                    <l n="5">ወእምኀበ፡ ሀሎ፡ ትዕቢት፡ አምጣነ፡ ሀሎ፡ ሀሳር፤</l>
+                    <l n="6">ቅድመ፡ ጽዮን፡ እሳት፡ አማጺ፡ አጵሎን፡ ማዕረር፤</l><pb n="39r"/>
+                    <l n="7">ሕልመ፡ ሴዴ፡ ቅያስ፡ ኮነ፡ ለኵሉ፡ ኖላዊ፡ ሐዲስ፡ ደብር፤</l>
+                    <l n="8">ኢየሐድጋ፡ ለምድር፡ አኮኑ፡ እንበለ፡ ዋህድ፡ ሔር፤</l>
+                    <l n="9">አርኡት፡ ዘለፋ፡ ዘንጸውር፡ ወእምርእስ፡ ዚአነ፡ አቅልል፡ በስቡህ፡ ምስጢር።</l>
+                    <l n="10">ከመ፡ ይርሐቅ፡ ተሐውኮ፡ መሪር።</l>
```

Comment:
Yes, it seems to be 10 lines, and I cannot see to which type it can be assigned. Unless we decide that line 9 and 10 belong together. It will be then a very long verse, which is a bit strange, but It may happen; perhaps an indication to that is the four dot asterisk at the end of verse 9, not two dots with dashes above and below, as before.

Reply:
Can I leave it as uncategorized or do you prefer that we reinterpret the lines?

Reply:
I would reinterpret the lines, though with a comment expressing some uncertainty (perhaps you can observe, on other copies, if the copiest was always exact with marking the end of the verse. In my experience, (Ethiopian) scribes were usually accurate). Also, qene was actually received by scholars because it is a stricktly formalized poetry, with its conventions apparently shared by the entire country. So, qene should always fit in one of the types :)

Reply:
I wrote a comment. You can look if it's clear :)

Reply:
Another possibility might be to divide line 8 or 9 into two lines (keeping the rhyme in -r)